src/NHSD.BuyingCatalogue.Ordering.Contracts/IOrderService.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using NHSD.BuyingCatalogue.Ordering.Domain;
+
+namespace NHSD.BuyingCatalogue.Ordering.Contracts
+{
+    public interface IOrderService
+    {
+        Task<Order> GetOrder(CallOffId callOffId);
+
+        Task<IList<Order>> GetOrderList(Guid organisationId);
+
+        Task<Order> GetOrderSummary(CallOffId callOffId);
+
+        Task<Order> GetOrderCompletedStatus(CallOffId callOffId);
```

Comment:
A think better name for this might be something like `GetOrderForStatusUpdate`.

Reply:
Updated method name to `GetOrderForStatusUpdate`